Sources/Identity/CustomerInfoManager.swift

```diff
@@ -26,6 +26,16 @@ class CustomerInfoManager {
     private let transactionFetcher: StoreKit2TransactionFetcherType
     private let transactionPoster: TransactionPosterType
 
+    // Can't have these properties with `@available`.
+    // swiftlint:disable identifier_name
+    var _diagnosticsTracker: Any?
```

Comment:
Right! Since we have availability annotations in the `DiagnosticsTrackerType` functions and not in the type, we can do that. 

Thanks for the suggestion!